src/common/BaseAnnotator.ts

```diff
@@ -46,11 +52,13 @@ export default class BaseAnnotator extends EventEmitter {
 
     container: Container;
 
+    features: Features;
+
     intl: IntlShape;
 
     store: store.AppStore;
 
-    constructor({ apiHost, container, file, fileOptions, intl, token }: Options) {
+    constructor({ apiHost, container, features = {}, file, fileOptions, intl, token }: Options) {
```

Comment:
Should we handle this as `this.features = features || {}`, to prevent any other falsy values (null, for example)?